package/src/index.ts

```diff
@@ -70,6 +92,260 @@ export class CandleClient {
   }): Promise<string> {
     return this.candle.executeTool(tool);
   }
+
+  public async getAssetAccounts(
+    query: AssetAccountQuery = {
+      linkedAccountIDs: undefined,
+      assetKind: undefined,
+    }
+  ): Promise<AssetAccount[]> {
+    const accounts = await this.candle.getAssetAccounts(query);
+    return accounts.map(this.convertToAssetAccount);
+  }
+
+  public async getTrades(
+    query: {
+      gainedAssetKind?: TradeQueryAssetKind;
+      lostAssetKind?: TradeQueryAssetKind;
+      counterpartyKind?: "merchant" | "user" | "service";
+    } & TradeQuery = {
+      counterpartyKind: undefined,
+      gainedAssetKind: undefined,
+      lostAssetKind: undefined,
+      linkedAccountIDs: undefined,
+      dateTimeSpan: undefined,
+    }
+  ): Promise<
+    {
+      dateTime: string;
+      state: TradeState;
+      counterparty: Counterparty;
+      lost: TradeAsset;
+      gained: TradeAsset;
+    }[]
+  > {
+    const trades = await this.candle.getTrades(query);
+    return trades.map((t) => ({
```

Comment:
```suggestion
    return trades.map(({ dateTime, state, counterparty, lost, gained }) => ({
```
#pp no single-letter variable names, especially when you can just unpack them